x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -41,256 +34,41 @@ First take a look to the [**Development Best Practices**](#development-best-prac
+| cypress:run | Runs all tests tagged as ESS placed in the `e2e` directory excluding `investigations` and `explore` directories in headless mode |
+| cypress:run:serverless | Runs all tests tagged as SERVERLESS in the `e2e` directory excluding `investigations` and `explore` directories in headless mode |
 | cypress:run:cases | Runs all tests under `explore/cases` in the `e2e` directory related to the Cases area team in headless mode |
 | cypress:run:reporter | Runs all tests with the specified configuration in headless mode and produces a report using `cypress-multi-reporters` |
 | cypress:run:respops | Runs all tests related to the Response Ops area team, specifically tests in `detection_alerts`, `detection_rules`, and `exceptions` directories in headless mode |
-| cypress:run:ccs | Runs all tests in the `ccs_e2e` directory in headless mode |
-| cypress:run:upgrade | Runs all tests in the `upgrade_e2e` directory in headless mode |
-| cypress:investigations:run | Runs all tests in the `e2e/investigations` directory in headless mode |
-| cypress:explore:run | Runs all tests in the `e2e/explore` directory in headless mode |
+| cypress:investigations:run | Runs all tests tagged as ESS in the `e2e/investigations` directory in headless mode |
+| cypress:explore:run | Runs all tests tagged as ESS in the `e2e/explore` directory in headless mode |
+| cypress:investigations:run:serverless | Runs all tests tagged as SERVERLESS in the `e2e/investigations` directory in headless mode |
+| cypress:explore:run:serverless | Runs all tests tagged as SERVERLESS in the `e2e/explore` directory in headless mode |
 | junit:merge | Merges individual test reports into a single report and moves the report to the `junit` directory |
 
 Please note that all the headless mode commands do not open the Cypress UI and are typically used in CI/CD environments. The scripts that open the Cypress UI are useful for development and debugging.
 
-### Execution modes
-
-There are currently four ways to run the tests, comprised of two execution modes and two target environments, which will be detailed below.
-
-#### Interactive mode
-
-When you run Cypress in interactive mode, an interactive runner is displayed that allows you to see commands as they execute while also viewing the application under test. For more information, please see [cypress documentation](https://docs.cypress.io/guides/core-concepts/test-runner.html#Overview).
-
-#### Headless mode
-
-A headless browser is a browser simulation program that does not have a user interface. These programs operate like any other browser, but do not display any UI. This is why meanwhile you are executing the tests on this mode you are not going to see the application under test. Just the output of the test is displayed on the terminal once the execution is finished.
-
```

Comment:
Nit: Removing all these sections from the README doesn't feel related to this PR (please correct me if I'm wrong). Could we push them in a separate PR with a description explaining the intent behind these changes?

Reply:
When updating the readme, to keep it simple I removed all the things that have been discontinued. All the code will be removed as well in further PRs, no problem at all to do what you are suggesting.